ntbs-service/Migrations/20191128192316_IndexNotificationStatusOnNotificationsTable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class IndexNotificationStatusOnNotificationsTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
+                name: "IX_Notification_NotificationStatus",
+                table: "Notification",
+                column: "NotificationStatus");

```

Comment:
I'm not sure this alone will fix issues- we might want an index over multiple columns, e.g.:
`Status, Id`
and /or 
`Status, Id, TbServiceId`
?